app/models/katello/root_repository.rb

```diff
@@ -407,7 +407,7 @@ def on_demand?
 
     def pulp_update_needed?
       changeable_attributes = %w(url unprotected checksum_type docker_upstream_name download_policy mirroring_policy verify_ssl_on_sync
```

Comment:
We don't need mirroring_policy either since it's passed in at sync time too:
```suggestion
      changeable_attributes = %w(url unprotected checksum_type docker_upstream_name download_policy verify_ssl_on_sync
```

Reply:
Added mirroring policy.. 👍🏼